### ISSUE TITLE
Audio stream being picked up in release_group

### ISSUE THREAD

**Issue by OmgImAlexis** [guessit-io/guessit#556]
```
For: Mr Robot - S03E01 - eps3 0 power-saver-mode h (1080p AMZN WEB-DL x265 HEVC 10bit EAC3 6.0 RCVR).mkv
GuessIt found: {
    "title": "Mr Robot",
    "season": 3,
    "episode": 1,
    "episode_title": "eps3 0 power-saver-mode h",
    "screen_size": "1080p",
    "streaming_service": "Amazon Prime",
    "source": "Web",
    "video_codec": "H.265",
    "color_depth": "10-bit",
    "audio_codec": "Dolby Digital Plus",
    "release_group": "6.0 RCVR",
    "container": "mkv",
    "mimetype": "video/x-matroska",
    "type": "episode"
}
```

**Comment by dremy7** [comment]
This has been an issue with guessit for forever. It only seems to recognize specific numbers of audio channels like 5.1 or 2.0. It's particularly bad with cirterion edition releases of old movies that have mono audio where the file is named FLAC1.0-RELEASEGRP. 1.0 audio channel is only parsed if the file name has it listed as MONO.

**Comment by ratoaq2** [comment]
Do you guys know that you can define these options when running guessit?

https://github.com/guessit-io/guessit/blob/develop/docs/configuration.rst

@Toilal I realised https://guessit.readthedocs.io/en/latest/
is not up-to-date

**Comment by ratoaq2** [comment]
Fixed: 315ae4d48236ba315319874efbc018eb7d03b43b
